includes/class-apple-news.php

```diff
@@ -1132,7 +1142,7 @@ protected function load_example_themes() {
 	}
 
 	/**
-	 * Verifies that the list of keys provided all exist in the settings array.
+	 * Verifies that the list of keys provided all exists in the settings array.
```

Comment:
I would go back to `exist` or maybe change it to `Verifies that the keys provided all exist in the settings array.` so there's less confusion.